client/me/help/help-courses/index.jsx

```diff
@@ -37,6 +37,17 @@ function getCourses() {
 			],
 			videos: [
 				{
+					date: i18n.moment( new Date( 'Thur, 2 Feb 2017 17:00:00 +0000' ) ),
+					title: i18n.translate( 'Everything You Need to Know About SEO and Marketing for Your Website' ),
+					description: i18n.translate(
+						'You will leave this 60-minute session with a plan for optimizing your site for search engines ' +
+						'and an overview of basic tactics for increasing your traffic. Our Happiness Engineers will provide a' +
+						'foundation for making sure that your site meets current SEO standards. We\'ll also discuss how to ' +
```

Comment:
Not sure why the a8ci18n bot didn't post here, but the string is missing a space between "a" and "foundation". See https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/help-course-videos/add-seo-video